book/installation.rst

```diff
@@ -304,6 +304,30 @@ several minutes to complete.
 
 .. _installing-a-symfony2-distribution:
 
+Installing the Symfony Demo Application
+---------------------------------------
+
+The Symfony Demo application is a fully-functional application that shows in
+practice the recommended way to develop Symfony applications. The application
```

Comment:
"in practice" feels a bit odd in this sentence. Maybe move it to the end of the sentence?